CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing Guide For Einstein-GPT-for-Developers
```

Comment:
Should `Einstein-GPT-for-Developers` be `salesforcedx-icons`, or is the product name the right thing?

Reply:
No, I think it should be salesforcedx-icons here.  copy pasta miss.  Good catch!